hknweb/polls/views.py

```diff
@@ -0,0 +1,4 @@
+from django.http import HttpResponse
+
+def index(request):
+    return HttpResponse("Hello, world. You're at the polls index.")
```

Comment:
Please add a new line for every file
Looks like you did and forgot this one

Reply:
Fixed for next PR